schema/schema.go

```diff
@@ -75,6 +76,8 @@ func (ta *Table) AddColumn(name string, columnType string, extra string) {
 				")"),
 			"'", "", -1),
 			",")
+	} else if "datetime" == columnType {
```

Comment:
@sputnik13 
I think we should use prefix to check this, if the time supports precision, `desc` tables may like:
```
CREATE TABLE t1 (t TIME(3), dt DATETIME(6));

desc t1;
+-------+-------------+------+-----+---------+-------+
| Field | Type        | Null | Key | Default | Extra |
+-------+-------------+------+-----+---------+-------+
| t     | time(3)     | YES  |     | NULL    |       |
| dt    | datetime(6) | YES  |     | NULL    |       |
+-------+-------------+------+-----+---------+-------+
```